docs/persistence.md

```diff
@@ -40,9 +40,8 @@ When run tests and CLI command locally, Cadence by default uses a user `uber` wi
 You can use the following command to create user(role) and grant access. 
 In the mysql shell:
 ```
-> CREATE USER 'uber' IDENTIFIED BY 'uber';
-> GRANT All Privileges to 'uber';
-
+> CREATE USER 'uber'@'%' IDENTIFIED BY 'uber';
+> GRANT ALL PRIVILEGES ON *.* TO 'uber'@'%';
```

Comment:
It should be fine but just want to check, do you know what's the differences? It seems like All is more than *.*

Reply:
And also what difference it makes for `'uber'@'%'`?

Reply:
1. `GRANT ALL PRIVILEGES TO 'uber'@'%';` without `ON *.*` doesn't work on MySQL8.0 via docker, I know I should've used 5.7. For people didn't read all the full documentation, `docker pull mysql` will pull `8.*` version.
2. `The host name part of the account name, if omitted, defaults to '%'.`, I just copied them from [/docs/setup/MYSQL_SETUP.md](/docs/setup/MYSQL_SETUP.md) and [/docker/buildkite/mysql-init/init.sql](/docker/buildkite/mysql-init/init.sql), so they look the same.